packages/docs/docs/lambda/lambdarolepermissions.md

```diff
@@ -0,0 +1,102 @@
+---
+image: /generated/articles-docs-lambda-lambda-role-permissions.png
+id: lambdarolepermissions
+title: Lambda permissions
+slug: /lambda/lambda-role-permissions
+crumb: "Lambda"
+---
+
+import {UserPolicy} from '../../components/lambda/user-permissions.tsx';
+
+## Context
+
+As documented on [`permission`](/docs/lambda/permissions) we create a remotion user and assign `policies`, this policies gives permissions to the `lambda` to render video from [`renderMediaOnLambda()`](/docs/lambda/rendermediaonlambda). This approach is sometimes not acceptable as AWS client id and secret associated with the user are considered long term credentials and prone for inadvertent leakage.
```

Comment:
We'll go over and improve some grammatical things in these sections - @MehmetAdemi can you help with improving the language in this document?